backend/.env.example

```diff
@@ -0,0 +1,7 @@
+DB_NAME=dbnamegoeshere
+DB_USER=usernamegoeshere
+DB_PASSWORD=dbpasswordgoeshere
```

Comment:
secrets leaked, please add this file to the .gitignore

Reply:
if you look at the values you will see that they are placeholders
so the file is to help someone who pulls the repo to use it to create the real `.env` file